recipes-bsp/tegra-binaries/tegra-nvpmodel_32.2.3.bb

```diff
@@ -17,6 +17,7 @@ do_compile[noexec] = "1"
 NVPMODEL = ""
 NVPMODEL_tegra186 ?= "nvpmodel_t186"
 NVPMODEL_tegra194 ?= "nvpmodel_t194"
+NVPMODEL_xavier-8gb ?= "nvpmodel_t194_8gb"
```

Comment:
Any reason you couldn't just have used `jetson-xavier-8gb` as the override here?

Reply:
This serves to identify the SOM regardless of the MACHINE variable. A customizing layer will set a new MACHINE to identify a unique base board, but we still need this override to apply.

If you have a better idea for the name that identifies the SOM I’m not married to this. I’ll just blame NVIDIA marketing for the lazy naming that doesn’t attempt to separate out modules and their own base boards :)

Reply:
OK, makes sense. "xavier-8gb" is certainly more understandable than "galen-8gb" or "p2888-0006".